process/process.go

```diff
@@ -0,0 +1,33 @@
+package process
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+// IncludeFile replaces intsances of <<include(file)>> with the contents
+// of "file", escaping instances of "<<" within the file before returning.
+func IncludeFile(template string, root string) (string, error) {
```

Comment:
Naming nits:

`IncludeFile` -> `MaybeIncludeFile` (since 99% of the time it's going to be called with a string that doesn't have an `include` statement

`template` -> ... not sure. `s` is a lot vaguer but I think less confusing than `template` because it is definitely a string but not necessarily a template. `maybeTemplate`? Not sure.

`root` -> `orbDirectory` or `orbRootDirectory` or something like that - I think this would be more clear